### PR TITLE
Rename package

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

**Review by anew — COMMENTED** [review]
- In Readme.md there is still a reference to Cask. 
- Also in market...spec.json, better use CDAP for the author and org

File: market/plugin-data-profiler/1.0.0/spec.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
   "specVersion": "1.0",
   "label": "Data Profiler",
   "description": "Generate statistics for each column or field within the data feed. Supports logical, informational, quantitative, uniques statistics generation",
-  "author": "Cask",
-  "org": "Cask Data, Inc.",
+  "author": "CDAP",
+  "org": "CDAP",
   "created": 1510006834,
```
Comment (anew) [inline market/plugin-data-profiler/1.0.0/spec.json:7]: should this be updated? Or will that be part of the marketplace P?

**Review by anew — APPROVED** [review]

File: market/plugin-data-profiler/1.0.0/spec.json (hunk 1 above)
Comment (elfenheart) [reply market/plugin-data-profiler/1.0.0/spec.json:7]: It will be part of market

Comment (elfenheart) [reply market/plugin-data-profiler/1.0.0/spec.json:7]: It will be part of market

**Pull request merged**